### PR TITLE
home is updated

### PR DESCRIPTION
**Pull request by GozdeUnver**
I have added buttons and paths of exchangeRaeAPI, airquality, core and randomActivity to the home app, modified settings.py so that all apps are stated there correctly but nbaStats, sport_relation, core apps are not working properly. sport_relation is using the html files of nbastats and core app is not working at all because there isn't any paths to go like this: "localhost/8000/core". I will talk to @e7hdk to look at his app, core

### CODE REVIEW

File: practice-app/group5_api/group5_api/urls.py [hunk 1]
```diff
@@ -16,8 +16,6 @@
 from django.urls import path,include
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
+    #path('admin/', admin.site.urls),
```
Comment (atasoyfurkan) [inline practice-app/group5_api/group5_api/urls.py:19]: I resolve the problem with admin page. The reason was name spacing issue (same name in different app). I solve it with adding app name to urls.py for core application.

**Review by atasoyfurkan — CHANGES_REQUESTED** [review]
You should not include pycache file.

**Comment by atasoyfurkan** [comment]
While reviewing, I and Gozde found a couple of bug in the system. I resolved them one by one. They are:
- 5 app did not have correct template order. It has to be /templates/app_name/some.html. I change to correct by this way, bugs are resolved.
- Core app did not have root '/' view I change /authenctication to root '/' so problem is resolved.
- Core app did have name spacing issue. That was causing to error in the main admin app so I add name spacing to Core app.
- requirements.txt had wrong requirement "decouple" I change it with correct one "python-decouple".

I am pushing changes to home branch.

**Comment by GozdeUnver** [comment]
Furkan will create a new pull request since he pulled this branch and made the modifications on it, he will make a new push and I will review it

**Pull request closed without merging**